tools/releaseToWinget.ps1

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+param(
+    [Parameter(Mandatory)]
+    [semver]
+    $ReleaseVersion,
+
+    [Parameter()]
+    [string]
+    $WingetRepoPath = "$PSScriptRoot/../../winget-pkgs",
+
+    [Parameter()]
+    [string]
+    $FromRepository = 'rjmholt',
+
+    [Parameter()]
+    [string]
```

Comment:
Maybe make this mandatory

Reply:
If you run it yourself with PowerShellForGitHub and don't provide a token, it will use its locally stored token